src/components/calculator.css

```diff
@@ -0,0 +1,60 @@
+.main { /* Main container */
+  width: 100%;
+  max-width: 300px;
+  height: auto;
```

Comment:
- [ ] As requested by the previous reviewer please fix these styles for your app. Use the correct color for the buttons and make sure it matches with the design.
### Your design

![image](https://user-images.githubusercontent.com/83803414/164451438-93c3c8a1-1ba3-4f26-8e0d-5d2339748d12.png)

### Given design

![image](https://user-images.githubusercontent.com/77190203/164444559-96b23790-20c2-44e4-bfa2-e41069aae2af.png)